examples/tree_2d_dgsem/elixir_euler_kelvin_helmholtz_instability_no_shock_capturing.jl

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Isn't this one relatively unstable? IF so, I would prefer not to add it (since we can get the same setup via `trixi_include(joinpath(examples_dir(), "tree_2d_dgsem", "elixir_euler_kelvin_helmholtz_instability.jl"), volume_integral=VolumeIntegralFluxDifferencing(flux_ranocha))`.

Reply:
Yes, it is - I'll remove it in the next round of PRs. Thanks!